### ISSUE TITLE
createObjectURL to avoid tainted canvas

### ISSUE THREAD

**Issue by jakob-e** [m90/seeThru#40]
To avoid error when using getImageData (`The canvas has been tainted by cross-origin data`)
consider adding source as objectURL:  

````JavaScript
//  HTML: <video playsinline muted autoplay loop></video>
function loadAsObjectURL(video, url){
  const xhr = new XMLHttpRequest();    
  xhr.responseType = 'blob';
  xhr.onload = (response) => video.src = URL.createObjectURL(xhr.response);
  xhr.onerror = () => { /* Houston we have a problem */}; 
  xhr.open('GET', url, true);
  xhr.send();
  video.onload = () => URL.revokeObjectURL(target.src);
}

const video = document.querySelector('video');    
video.addEventListener('loadedmetadata', ()=> {
  seeThru.create(video);    
});
loadAsObjectURL(video, '../video.mp4');


`````
Test [https://codepen.io/jakob-e/pen/RZypBp](https://codepen.io/jakob-e/pen/RZypBp)

**Comment by m90** [comment]
Thanks for the info. Would you mind opening a PR with that info against the README (just like for #39)? That's be super helpful and much appreciated.

**Comment by m90** [comment]
Solved via #41
